source/faq.rst

```diff
@@ -1,264 +1,219 @@
 
 .. _sys-req:
 
+*************************************************
 What are the minimum system requirements for PMM?
-================================================================================
+*************************************************
 
-.. rubric:: |pmm-server|
+.. rubric:: PMM Server
 
 Any system which can run Docker version 1.12.6 or later.
 
-It needs roughly 1 GB of storage for each monitored database node
-with data retention set to one week.
+It needs roughly 1 GB of storage for each monitored database node with data retention set to one week.
+
+.. note::
+
+   By default, :ref:`retention <data-retention>` is set to 30 days for Metrics Monitor and for Query Analytics.  You can consider :ref:`disabling table statistics <performance-issues>` to decrease the Prometheus database size.
+
+The minimum memory requirement is 2 GB for one monitored database node.
 
 .. note::
 
-   By default, :ref:`retention <data-retention>` is set to 30 days for
-   Metrics Monitor and for Query Analytics.  Also consider
-   :ref:`disabling table statistics <performance-issues>`, which can
-   greatly decrease Prometheus database size.
+   The increase in memory usage is not proportional to the number of nodes.  For example, data from 20 nodes should be easily handled with 16 GB.
+
+.. rubric:: PMM Client
 
-Minimum memory is 2 GB for one monitored database node, but it is not
-linear when you add more nodes.  For example, data from 20 nodes
-should be easily handled with 16 GB.
+Any modern 64-bit Linux distribution. It is tested on the latest versions of Debian, Ubuntu, CentOS, and Red Hat Enterprise Linux.
 
-.. rubric:: |pmm-client|
+A minimum of 100 MB of storage is required for installing the PMM Client package.  With a good connection to PMM Server, additional storage is not required.  However, the client needs to store any collected data that it cannot dispatch immediately, so additional storage may be required if the connection is unstable or the throughput is low.
 
-Any modern 64-bit Linux distribution. It is tested on the latest
-versions of Debian, Ubuntu, CentOS, and Red Hat Enterprise Linux.
 
-Minimum 100 MB of storage is required for installing the |pmm-client|
-package.  With good constant connection to |pmm-server|, additional
-storage is not required.  However, the client needs to store any
-collected data that it is not able to send over immediately, so
-additional storage may be required if connection is unstable or
-throughput is too low.
 
 .. _data-retention:
+.. _how-to-control-data-retention-for-pmm:
 
+**************************************
 How to control data retention for PMM?
-================================================================================
+**************************************
 
-By default, both |prometheus| and QAN store time-series data for 30 days.
+By default, both Prometheus and QAN store time-series data for 30 days.
```

Comment:
I thought we stopped using QAN in favour of Query Analytics?

Reply:
@rnovikovP Is that true?

Reply:
Why on earth do we need 2 objects/terms here if this is PMM who stores data?  there is no option to set the different retention periods for them separately. 
As for  QAN vs Query Analytics  - I'll belive Michael in this case )) it's better to avoid TLA  (Three-letter acronym)

Reply:
No idea why both Prometheus and Query Analytics are mentioned in this context. Perhaps it was to explain where the data is stored, by which components. I'll rephrase it to just say PMM.

Reply:
it was there because previously we had two different variables to ser retention for Prom (30 days by default ) and for QAN (7 days by default )